pep-0567.rst

```diff
@@ -551,6 +604,53 @@ code unmodified, but will automatically enable support for
 asynchronous code.
 
 
+Examples
+========
+
+Converting code that uses threading.local()
+-------------------------------------------
+
+A typical code that uses ``threading.local()`` usually looks like
+the following snippet::
+
+    class mylocal(threading.local):
+        # Subclass threading.local to specify a default value.
+        value = 'spam'
+
+    mylocal = mylocal()
+
+    # To set a new value:
+    mylocal.value = 'new value'
+
+    # To read the current value:
+    mylocal.value
+
+
+Such code can be converted to use the ``contextvars`` module::
+
+    mylocal = contextvars.ContextVar('mylocal', 'spam')
+
+    # To set a new value:
+    mylocal.set('new value')
+
+    # To read the current value:
+    mylocal.get()
+
+
+Offloading execution to other threads
+-------------------------------------
+
+It is possible to run code in a separate OS thread using a copy
+of the current thread context::
+
+    executor = ThreadPoolExecutor()
+    current_context = contextvars.copy_context()
+
+    executor.submit(
+        lambda: current_context.run(some_function))
```

Comment:
Since `submit()` takes both a callable and optional args, this can be simplified to
```py
executor.submit(current_context.run, some_function)
```

Reply:
fixed